mentees/README.md

```diff
@@ -0,0 +1,264 @@
+## **Become a Mentee -- Draft**
+
+Congratulations on choosing to take part in a mentorship program available with the CNCF and some of its valued members. Contributing as a mentee can be a rewarding and fulfilling experience that connects you with a range of experts and development opportunities unique to these programs. 
+
+In addition to providing a high-quality learning experience that can help to kickstart your career, while adding tangible value to one of CNCF's 140+ open source projects; participating in mentorship programs also offer a range of other benefits that help both yourself and the CNCF community:
+* **Community:** Your contribution matters! Participating in mentorship programs can help to strengthen the community and its work by fostering innovation and collaboration while improving overall project quality.
+* **Network:** Learn and work alongside experts developing some of the most influential open source technologies in the world, as well as other passionate mentees eager to help bring these projects to life. 
+* **Recognition:** Many programs offer stipends and other incentives to support your contribution. Amounts can be based on varying factors including your location or specific projects. 
+
+Becoming a mentee in a CNCF mentorship program provides numerous benefits, including access to experienced professionals in the field, opportunities for personal and professional growth, and a supportive network for guidance and feedback. It can also help you to acquire new skills and knowledge, develop a deeper understanding of the industry, and increase your chances of success in your career.
+
+
+## **Mentee Expectations**
+Participating in a mentorship program can require a lot of time, effort and dedication. In exchange for that commitment, a seasoned mentor offers their valuable time and expertise to aid your learning and development. 
+
+To ensure the experience is positive for everyone involved, there are a few key things you as a mentee can do to maximise your opportunity:
+
+* **Make sure you can devote yourself to the minimum required hours throughout the program.** This can vary between projects, but often you will be expected to invest a significant amount of time on a weekly basis to your work. This might also mean taking priority over your study or personal obligations, depending on the project. 
```

Comment:
> This might also mean taking priority over your study or personal obligations

That doesn't sit right with me. I would never tell someone that they should put a mentoring program over their family or completing their degree. Is that what is really meant?

Reply:
Good catch, I missed that. I agree, this is not a recommendation we should make. We should be flexible (especially in programs like LFX Mentoring where we have control of dates and are able to make exceptions on an individual basis)